app/services/course/assessment/answer/multiple_response_auto_grading_service.rb

```diff
@@ -41,9 +41,9 @@ def grade_any_correct(question, answer)
   def grade_all_correct(question, answer)
     correct_answers = question.options.correct
     correct_selection = correct_answers & answer.options
-    correct = correct_selection.length == correct_answers.length
+    answer.correct = correct_selection.length == correct_answers.length
 
-    [grade_for(question, correct), explanations_for(answer.options)]
+    [grade_for(question, answer.correct), explanations_for(answer.options)]
```

Comment:
I think I put this in the wrong place, the correct should be a value returned by concrete grading service (same as grade and explanations).  Will update.